addons/website/static/tests/tours/background_colorpicker.js

```diff
@@ -0,0 +1,69 @@
+/** @odoo-module **/
+
+import wTourUtils from "@website/js/tours/tour_utils";
```

Comment:
It s probably easier to do a unit test in js. Or I miss something ? It s strange to do a test in website to fix something in web.

Reply:
Hello @FrancoisGe i believe though the changes are made in `colorpicker.js` but the functionality these changes have brought is visible in editor or website where the color picker is shown, and in my opinion the test too should be written there only.

Reply:
@FrancoisGe We tried to write the qunit test case but with backend we were not able to produce such case and hence we have written tour for the same.

Reply:
Sorry I didn't see your ping. If I don't answer after 2 days, you can ping me directly on discord :) 

Mmh, in 18.0 we should write a qunit test in community/addons/html_editor/static/tests/color_selector.test.js.
Ok for the tour in 17. (The best solution, it s probably to create a test file specific for ColorPicker)

> Hello @FrancoisGe i believe though the changes are made in `colorpicker.js` but the functionality these changes have brought is visible in editor or website where the color picker is shown, and in my opinion the test too should be written there only.

Reply:
@FrancoisGe, We will write QUnit tests in version 18.0. For now, we have created a separate file for the test tour, named ``colorpicker.js``